src/lib/i18n/texts/de.json

```diff
@@ -66,22 +66,22 @@
       "losses": "Niederlagen",
```

Comment:
Probably, yes. I wasn't familiar with tha use of bye. It basically says „Wins while outnumbering the enemy“ whereas I understand “bye“ to be a win due to the enemy not showing up. Not sure if that is supposed to be the same in GW2 context.